### PR TITLE
Update history page urls for beta and alpha registers

### PR DESCRIPTION
**Pull request by danmitchell-**
- Register
- Field
- Datatype
- Country
- Territory
- Local authority type
- Local authority england
- School England
- Notifiable animal disease

### CODE REVIEW

File: ansible/group_vars/tag_Environment_beta [hunk 1]
```diff
@@ -2,26 +2,30 @@ register_domain: register.gov.uk
...
   datatype:
     custodian_name: Paul Downey
+    history_page_url: https://register-status.herokuapp.com/registers/datatype
   field:
     custodian_name: Paul Downey
+    history_page_url: https://register-status.herokuapp.com/registers/field
   local-authority-eng:
     custodian_name: Stephen McAllister
-    history_page_url: https://registers-history.herokuapp.com/local-authority-eng
+    history_page_url: https://register-status.herokuapp.com/registers/local-authority-england
```
Comment (samcrang) [inline ansible/group_vars/tag_Environment_beta]: Are we okay with naming not being consistent between the register and the history page?

File: ansible/group_vars/tag_Environment_alpha [hunk 2]
```diff
@@ -27,10 +28,9 @@ register_settings:
     registers_yaml_location: "s3://openregister.alpha.config/registers.yaml"
   school-eng:
     custodian_name: Andrew Thomson
-    history_page_url: https://registers-history.herokuapp.com/school-eng
+    history_page_url: https://register-status.herokuapp.com/registers/school-england
```
Comment (samcrang) [inline ansible/group_vars/tag_Environment_alpha]: I think this should be `school-eng` rather `school-england`?

Comment (danmitchell-) [inline ansible/group_vars/tag_Environment_alpha]: Good spot.

File: ansible/group_vars/tag_Environment_beta (hunk 1 above)
Comment (danmitchell-) [inline ansible/group_vars/tag_Environment_beta]: Fixed

Comment (samcrang) [inline ansible/group_vars/tag_Environment_beta]: nice m8, nice

**Pull request merged**